### PR TITLE
Remove any selinux policy packages

### PR DESCRIPTION
**Pull request by mkolesnik**
We're not using selinux in the container, remove any policy package to
save some space..

This specifically shaves off some 30MB :)

### CODE REVIEW

File: package/Dockerfile.dapper-base [hunk 1]
```diff
@@ -39,7 +39,7 @@ RUN dnf -y distrosync --nodocs --setopt=install_weak_deps=False && \
                    gcc git-core curl moby-engine make golang kubernetes-client \
                    findutils upx jq golang-x-tools-goimports ShellCheck shflags && \
     dnf -y remove libsemanage && \
-    rpm -e --nodeps selinux-policy-targeted && \
+    rpm -qa "selinux*" | xargs rpm -e --nodeps && \
```
Comment (skitt) [inline package/Dockerfile.dapper-base:42]:
```suggestion
    rpm -qa "selinux*" | xargs rpm -r -e --nodeps && \
```

so this doesn’t fail if `rpm -qa` doesn’t output anything ;-).

Comment (mkolesnik) [reply package/Dockerfile.dapper-base]: Are you sure it's `-r`?

It seems to fail with
```
rpm: arguments to --root (-r) must begin with a /
```

Comment (skitt) [reply package/Dockerfile.dapper-base]: Oops, sorry, wrong place, it’s meant for `xargs`: `xargs -r rpm -e --nodeps && \`

File: package/Dockerfile.dapper-base [hunk 2]
```diff
@@ -39,7 +39,7 @@ RUN dnf -y distrosync --nodocs --setopt=install_weak_deps=False && \
                    gcc git-core curl moby-engine make golang kubernetes-client \
                    findutils upx jq golang-x-tools-goimports ShellCheck shflags && \
     dnf -y remove libsemanage && \
-    rpm -qa "selinux*" | xargs rpm -e --nodeps && \
+    rpm -qa "selinux*" | xargs rpm -r -e --nodeps && \
```
Comment (skitt) [inline package/Dockerfile.dapper-base:42]:
```suggestion
    rpm -qa "selinux*" | xargs -r rpm -e --nodeps && \
```

Sorry, this is the correct invocation.

Comment (mkolesnik) [reply package/Dockerfile.dapper-base]: OK we'll just squash it before merge :)

**Pull request merged**